code/game/gamemodes/antag_datum.dm

```diff
@@ -0,0 +1,197 @@
+/mob/living
+	var/list/antag_datums
```

Comment:
On the other hand does blob need to have an antagonist datum? It probably does...
On the other hand we could totally cheat with the design, it really depends on how we want to track these.